### PR TITLE
[1LP][RFR] Fixed test_provisioning_dialog_crud

### PR DESCRIPTION
**Pull request by quarckster**
{{pytest: cfme/tests/automate/test_provisioning_dialogs.py::test_provisioning_dialog_crud -v}}

### CODE REVIEW

File: cfme/tests/automate/test_provisioning_dialogs.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def test_provisioning_dialog_crud(appliance):
     assert dialog.exists
 
     with update(dialog):
-        dialog.diag_type = collection.HOST_PROVISION
+        dialog.diag_type = collection.VM_MIGRATE
```
Comment (sshveta) [inline cfme/tests/automate/test_provisioning_dialogs.py]: Question : Can you pls comment why is this change needed ? 
Host_provision is still an option for provisioning dialogs.

Comment (quarckster) [reply cfme/tests/automate/test_provisioning_dialogs.py]: It seems it's not anymore in CFME 5.10
